test_manifest_idempotent is failing

https://github.com/web-platform-tests/wpt/pull/14887 shows that test_manifest_idempotent in test_manifest.py is failing:
https://github.com/web-platform-tests/wpt/blob/de71007a5f31bcaf7ec4651584a851edd6bdbc8f/tools/manifest/tests/test_manifest.py#L84-L97

The failure log:
```
=================================== FAILURES ===================================
___________________________ test_manifest_idempotent ___________________________
    @h.given(hs.lists(sourcefile_strategy(),
>                     min_size=1, unique_by=lambda x: x.rel_path))
manifest/tests/test_manifest.py:85: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
.tox/py27/lib/python2.7/site-packages/hypothesis/core.py:583: in execute
    % (test.__name__, text_repr[0])
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <hypothesis.core.StateForActualGivenExecution object at 0x7fdbc3e5d910>
message = "Hypothesis test_manifest_idempotent(s=[<Mock id='140581614463120'>]) produces unreliable results: Falsified on the first call but did not on a subsequent one"
    def __flaky(self, message):
        if len(self.falsifying_examples) <= 1:
>           raise Flaky(message)
E           Flaky: Hypothesis test_manifest_idempotent(s=[<Mock id='140581614463120'>]) produces unreliable results: Falsified on the first call but did not on a subsequent one
.tox/py27/lib/python2.7/site-packages/hypothesis/core.py:748: Flaky
---------------------------------- Hypothesis ----------------------------------
Falsifying example: test_manifest_idempotent(s=[<Mock id='140581614463120'>])
Unreliable test timings! On an initial run, this test took 255.72ms, which exceeded the deadline of 200.00ms, but on a subsequent run it took 0.46 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.
You can reproduce this example by temporarily adding @reproduce_failure('4.0.1', 'AXicJYqxDQAwCMMCiCUImPr/q41aL5aVmB/iYQCRXghZxUEilz2ltf8JWxYjXxkHAOg=') as a decorator on your test case
```

This might have been introduced by https://github.com/web-platform-tests/wpt/pull/14890, or the problem was already there, it's not clear.

@jgraham you wrote this test, can you check if you understand the failure? It's presumably a flaky failure because of how hypothesis works.

--- a/tools/conftest.py
+++ b/tools/conftest.py
@@ -6,8 +6,10 @@ from hypothesis import settings, HealthCheck
 impl = platform.python_implementation()
 
 settings.register_profile("ci", settings(max_examples=1000,
+                                         deadline=None,
                                          suppress_health_check=[HealthCheck.too_slow]))
-settings.register_profile("pypy", settings(suppress_health_check=[HealthCheck.too_slow]))
+settings.register_profile("pypy", settings(deadline=None,
+                                           suppress_health_check=[HealthCheck.too_slow]))
 
 settings.load_profile(os.getenv("HYPOTHESIS_PROFILE",
                                 "default" if impl != "PyPy" else "pypy"))

